apps/gruenbuch/migrations/0002_auto_20211006_1029.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 3.2.7 on 2021-10-06 10:29
```

Comment:
With migrations, we try to make as few as possible, but if it helps the understanding and cleaner commits, then it's also fine to have more than one in one PR.
And we name them after what they do. You can just change the name between the `number_` and `.py`. So, you could call this `0002_add_context.py`. 
(But on the other hand wagtail is always producing lots of migrations, where it's often not clear, why they are there. So, here you can also just not do the above things. :smile: )
And remember: when you change your migrations afterwards, you need a new DB (or should make a backup of your DB before you run migrations you might want to change). As soon as migrations are merged, they shouldn't be changed anymore.